kraken-java-sdk/kraken-java-sdk-controller/src/test/java/com/consoleconnect/kraken/operator/controller/BuyerCreator.java

```diff
@@ -0,0 +1,45 @@
+package com.consoleconnect.kraken.operator.controller;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+import com.consoleconnect.kraken.operator.controller.dto.BuyerAssetDto;
+import com.consoleconnect.kraken.operator.controller.dto.CreateBuyerRequest;
+import com.consoleconnect.kraken.operator.core.model.HttpResponse;
+import com.consoleconnect.kraken.operator.core.toolkit.JsonToolkit;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+public interface BuyerCreator {
+  String PRODUCT_ID = "product.mef.sonata.api";
+  String BUYER_BASE_URL = String.format("/products/%s/buyers", PRODUCT_ID);
+  String BUYER_ID = "consolecore-poping-company";
```

Comment:
Replace the name with a mock one.

Reply:
Thanks. The test data  has been renamed.